examples/BUILD.bazel

```diff
@@ -50,6 +50,22 @@ java_grpc_library(
     deps = [":route_guide_java_proto"],
 )
 
+proto_library(
```

Comment:
I believe you can depend on the canonical build defs: 

https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/BUILD.bazel


I think you need to add this to the WORKSPACE file, and then reference it below.

Reply:
@carl-mastrangelo with the changes I made does this still apply? I have been able to get it to build without WORKSPACE file changes.